apps/oc_chef_wm/src/chef_wm_malformed.erl

```diff
@@ -47,6 +48,8 @@ malformed_request_message(bad_clock, Req, State) ->
     Msg = iolist_to_binary([<<"Failed to authenticate as ">>, User,
                             <<". Synchronize the clock on your host.">>]),
     {[{<<"error">>, [Msg]}]};
+malformed_request_message(bad_date, _Req, _State) ->
```

Comment:
To go with the earlier comment, {bad_date, FieldName} would let us make a nicer message.